stream-chat-android-client/src/main/java/io/getstream/chat/android/client/extensions/StateFlowExtensions.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:JvmName("StateFlowExtensions")
+
+package io.getstream.chat.android.client.extensions
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.asLiveData
+import kotlinx.coroutines.flow.StateFlow
+
+/**
+ * Creates a [LiveData] that has values collected from the original [StateFlow].
+ */
+@Suppress("NEWER_VERSION_IN_SINCE_KOTLIN")
+@SinceKotlin("99999.9")
+public fun <T> StateFlow<T>.asLiveData(): LiveData<T> = this.asLiveData()
```

Comment:
```suggestion
public fun <T> Flow<T>.asLiveData(): LiveData<T> = this.asLiveData()
```
`Flow` is broader than `StateFlow`

Reply:
good catch, thanks!